src/seedsigner/models/psbt_parser.py

```diff
@@ -277,16 +277,15 @@ def _get_policy(scope, scriptpubkey, xpubs):
             elif "p2sh" in script_type and scope.redeem_script is not None:
```

Comment:
Does this need to be a `"p2sh" == script_type` for native segwit single-sig?  I think this was corrected in 572, making sure nothing snuck back in.